src/status_im2/contexts/chat/messages/pin/banner/view.cljs

```diff
@@ -5,7 +5,15 @@
 (defn banner
   [chat-id]
   (let [pinned-messages (rf/sub [:chats/pinned chat-id])
-        latest-pin-text (get-in (last (vals pinned-messages)) [:content :text])
+
+        latest-pin-id   (-> pinned-messages
+                            vals
+                            last
+                            (get :message-id))
+        latest-pin-text (-> [:chats/chat-messages chat-id]
```

Comment:
The thread first macro here makes the code more difficult to comprehend. You can just use get-in once, and as usual, consider having a separate subscription to keep the UI dumb.

```clojure
(get-in (rf/sub [:chats/chat-messages chat-id])
        [latest-pin-id :content :text])
```